.install/layer-macro.yaml

```diff
@@ -0,0 +1,167 @@
+        - arn:aws:iam::aws:policy/ReadOnlyAccess
+      Policies:
+        - PolicyName: ECSComposeXAwsAccess
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Sid: CloudFormationAccess
+                Effect: Allow
+                Action:
+                  - cloudformation:ValidateTemplate
+                Resource:
+                  - "*"
+              - Sid: S3ObjectsAccess
+                Effect: Allow
+                Action:
+                  - s3:PutObject*
+                Resource: !If
+                  - AnyBucket
+                  - - "*"
+                  - - !Sub "arn:${AWS::Partition}:s3:::${TemplatesBucket}/*"
+
+  MacroLambdaFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      AutoPublishAlias: live
+      Description: Lambda function to execute ECS Compose-X CFN Macro
+      MemorySize: !Ref MemorySize
+      Role: !GetAtt 'MacroFunctionRole.Arn'
+      Timeout: 30
+      Runtime: python3.8
+      FunctionName: !Sub "ecs-compose-x-macro-${AWS::Region}"
+      Handler: index.lambda_handler
+      Layers:
+        - !Ref ComposeXLambdaLayerVersion
+      InlineCode: |
+        #!/usr/bin/env python
+        from ecs_composex.macro import lambda_handler as macro_function
+
+
+        def lambda_handler(event, context):
+            """ Entry point function """
+            return macro_function(event, context)
+
+      Tags:
+        Name: !Sub "ecs-compose-x-macro-${AWS::Region}"
+        Usage: ECS Compose-X CFN macro
+
+  CloudFormationLogRole:
+    Type: AWS::IAM::Role
+    Properties:
+      PermissionsBoundary: !If
+        - UseBoundary
+        - !Ref PermissionsBoundary
+        - !Ref AWS::NoValue
+      Tags:
+        - Key: Name
+          Value: CFN Macro Logging role
+        - Key: Usage
+          Value: ECS Compose-X
+      Policies:
+        - PolicyName: 'AllowLoggingForMacro'
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+              - Action:
+                  - logs:PutLogEvents
+                  - logs:CreateLogStream
+                Effect: Allow
+                Resource: [ !GetAtt 'CloudFormationLogGroup.Arn' ]
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Action:
+              - sts:AssumeRole
+            Effect: Allow
+            Principal:
+              Service:
+                - !Sub 'cloudformation.${AWS::URLSuffix}'
+
+  CloudFormationMacro:
+    Type: AWS::CloudFormation::Macro
+    Properties:
+      Description: Macro for ECS ComposeX embedded in CFN
+      FunctionName: !Ref MacroLambdaFunction
+      LogGroupName: !Ref CloudFormationLogGroup
+      LogRoleARN: !GetAtt 'CloudFormationLogRole.Arn'
+      Name: compose-x
+
+  CloudFormationLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub 'cloudformation/macro/compose-x'
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Fn::Sub isn&#39;t needed because there are no variables at Resources/CloudFormationLogGroup/Properties/LogGroupName/Fn::Sub